### DISCUSSION TITLE
Design the printed ticket layout

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3327] [Q&A] [answered]
Hello!

How can I design the layout of the ticket that a compatible thermal printer is printing ?
Not the pdf that a customer receives in his email, but the printed ticket.
Is this something that can be configured ?

**Comment by raphaelm** [comment]
In product settings, you can set different layouts per sales channel

**Reply by alemao8** [reply]
Do I have to install a payble plugin in order to print out tickets from the local thermal printer ?
I cannot see an option of offline sales like in the screenshot in https://marketplace.pretix.eu/products/offlinesales

If I design a layout in layout designer will this be printed in the thermal printer ?

**Reply by raphaelm** [reply]
You can design a layout in the thermal printer, then download tickets that have this layout assigned and print it through your computer on the thermal printer, no plugin required.

**Reply by alemao8** [reply]
where do I provide a ticket template for the offline ticket for Boca Lemur?
I refer to the type of ticket that is being printed to a Boca Lemur when I select it in the android app.

**Reply by alemao8** [reply]
In order to print a ticket with pretixPOS in BOX Office do I have to buy the offline sales plugin in https://marketplace.pretix.eu/products/offlinesales ?

**Reply by raphaelm** [reply, marked as answer]
No, offline sales plugin has nothing to do with this. If you use pretixPOS, you can configure the layout that should be used in the product settings in the backend
